Earthfile

```diff
@@ -19,7 +19,7 @@ test:
 setup:
   FROM +build
   RUN apt update && apt install xz-utils
-  RUN npx @replayio/playwright install
+  RUN npx --yes @replayio/playwright@latest install
```

Comment:
Why the change here? It should already use the version contained in the monorepo - which is like latest-latest 😅

Reply:
I was just trying out something. This command never exits: https://github.com/replayio/replay-cli/actions/runs/8395294816/job/22994255887?pr=327 I don't know if npx is being weird or there's something odd about the new version of node and npx.

Reply:
Or the version in monorepo is somehow mangled?

Reply:
I think it should be reverted as it most likely caught a legitimate issue here. I'm still digging into this but it's some kind of a leak~ with open handles. The node process doesn't ever exit when executing `npx @replayio/playwright install` and I can repro this locally. So far I managed to narrow it down to the fact that some write streams are still open:
<img width="935" alt="process._getActiveHandles() returning 2 WriteStreems" src="https://github.com/replayio/replay-cli/assets/9800850/9fcb4786-5573-4c4b-812d-665a3c8aa312">

Reply:
I pushed out the fix ( https://github.com/replayio/replay-cli/pull/327/commits/a4454e3d26e9b91124ed8d4ac7fba79101525d76 ) for this here. The code itself might be freely tweaked - I just pushed out a quick fix of sorts to validate that this actually fixes the problem.

The problem was that `@replayio/playwright` was importing a util from `@replayio/replay`. That led to its whole module being initialized and that included initializing the LaunchDarkly wrapper class (and the underlying LaunchDarkly client). Even though this wasn't *actually* used by anything in this scenario it was trying to periodically flush things here using a timer loop:
https://github.com/launchdarkly/js-sdk-common/blob/0c4a8c0f10cfd22c089b875f62fc0a2e0ba6c504/src/EventProcessor.js#L163-L169

That prevented the process from exiting. The `@replayio/playwright` package didn't have any logic to close that client since it's not concerned with it at all - it just wanted to import a util and that's it ;p